package-lock.json

```diff
@@ -41066,7 +41121,7 @@
 			}
 		},
 		"woocommerce": {
-			"version": "git+https://github.com/woocommerce/woocommerce.git#5213b05d57b0e0e703eaed3519694fa98a51516f",
+			"version": "git+https://github.com/woocommerce/woocommerce.git#16a289cefaf0c71d0ec4c9b8252ce5846973615c",
```

Comment:
This hash bump is causing issues, don't know why it keeps changing :|. It'll be nice when we can stop including woocommerce as a npm package and just import `woocommerce/e2e-utils` instead.